website/docs/docs/build/udfs.md

```diff
@@ -68,12 +68,13 @@ To define UDFs in dbt, refer to the following steps:
         config:
```

Comment:
^ above - i don't think description is required?

cc: @QMalcolm

Reply:
Description shouldn't be required